pynest/examples/EI_clustered_network/network.py

```diff
@@ -0,0 +1,807 @@
+            )
+
+        if self._params["I_th_I"] is None:
+            I_xI = self._params["I_xI"]
+        else:
+            I_xI = self._params["I_th_I"] * helper.rheobase_current(
+                self._params["tau_I"], self._params["E_L"], self._params["V_th_I"], self._params["C_m"]
+            )
+
+        E_neuron_params = {
+            "E_L": self._params["E_L"],
+            "C_m": self._params["C_m"],
+            "tau_m": self._params["tau_E"],
+            "t_ref": self._params["t_ref"],
+            "V_th": self._params["V_th_E"],
+            "V_reset": self._params["V_r"],
+            "I_e": I_xE
+            if self._params["delta_I_xE"] == 0
+            else I_xE * nest.random.uniform(1 - self._params["delta_I_xE"] / 2, 1 + self._params["delta_I_xE"] / 2),
+            "tau_syn_ex": self._params["tau_syn_ex"],
+            "tau_syn_in": self._params["tau_syn_in"],
+            "V_m": self._params["V_m"]
+            if not self._params["V_m"] == "rand"
+            else self._params["V_th_E"] - 20 * nest.random.lognormal(0, 1),
+        }
+        I_neuron_params = {
+            "E_L": self._params["E_L"],
+            "C_m": self._params["C_m"],
+            "tau_m": self._params["tau_I"],
+            "t_ref": self._params["t_ref"],
+            "V_th": self._params["V_th_I"],
+            "V_reset": self._params["V_r"],
+            "I_e": I_xI
+            if self._params["delta_I_xE"] == 0
+            else I_xI * nest.random.uniform(1 - self._params["delta_I_xE"] / 2, 1 + self._params["delta_I_xE"] / 2),
+            "tau_syn_ex": self._params["tau_syn_ex"],
+            "tau_syn_in": self._params["tau_syn_in"],
+            "V_m": self._params["V_m"]
+            if not self._params["V_m"] == "rand"
+            else self._params["V_th_I"] - 20 * nest.random.lognormal(0, 1),
+        }
+
+        # iaf_psc_exp allows stochasticity, if not used - ignore
+        if (self._params.get("delta") is not None) and (self._params.get("rho") is not None):
+            E_neuron_params["delta"] = self._params["delta"]
+            I_neuron_params["delta"] = self._params["delta"]
+            E_neuron_params["rho"] = self._params["rho"]
+            I_neuron_params["rho"] = self._params["rho"]
+
+        # create the neuron populations
+        pop_size_E = self._params["N_E"] // self._params["n_clusters"]
+        pop_size_I = self._params["N_I"] // self._params["n_clusters"]
+        E_pops = [
+            nest.Create(self._params["neuron_type"], n=pop_size_E, params=E_neuron_params)
+            for _ in range(self._params["n_clusters"])
+        ]
+        I_pops = [
+            nest.Create(self._params["neuron_type"], n=pop_size_I, params=I_neuron_params)
+            for _ in range(self._params["n_clusters"])
+        ]
+
+        self._populations = [E_pops, I_pops]
+
+    def connect(self):
+        """Connect the excitatory and inhibitory populations with each other
+        in the EI-clustered scheme
```

Comment:
Line breaks here and in other places seem strange. We allow 120 (don't we, @jessica-mitchell?), so the full line should fit. And should the first line begin with the `"""` or on the line below @jessica-mitchell ?

Reply:
@jessica-mitchell (https://github.com/nest/nest-simulator/pull/3028#pullrequestreview-1988394980) 
checked.